tmail-backend/integration-tests/rate-limiter/distributed-rate-limiter-integration-tests/src/test/java/com/linagora/tmail/integration/distributed/DistributedRateLimitingPlanIntegrationTest.java

```diff
@@ -0,0 +1,42 @@
+package com.linagora.tmail.integration.distributed;
+
+import org.apache.james.CassandraExtension;
+import org.apache.james.JamesServerBuilder;
+import org.apache.james.JamesServerExtension;
+import org.apache.james.mailbox.opendistro.DockerOpenDistroExtension;
+import org.apache.james.mailbox.opendistro.DockerOpenDistroSingleton;
+import org.apache.james.modules.AwsS3BlobStoreExtension;
+import org.apache.james.rate.limiter.RedisExtension;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.linagora.tmail.blob.blobid.list.BlobStoreConfiguration;
+import com.linagora.tmail.integration.RateLimitingPlanIntegrationContract;
+import com.linagora.tmail.james.app.DistributedJamesConfiguration;
+import com.linagora.tmail.james.app.DistributedServer;
+import com.linagora.tmail.james.app.RabbitMQExtension;
+import com.linagora.tmail.module.LinagoraTestJMAPServerModule;
+
+public class DistributedRateLimitingPlanIntegrationTest implements RateLimitingPlanIntegrationContract {
+    @RegisterExtension
+    static RedisExtension redisExtension = new RedisExtension();
+
+    @RegisterExtension
+    static JamesServerExtension testExtension = new JamesServerBuilder<DistributedJamesConfiguration>(tmpDir ->
+        DistributedJamesConfiguration.builder()
+            .workingDirectory(tmpDir)
+            .configurationFromClasspath()
+            .blobStore(BlobStoreConfiguration.builder()
+                .disableCache()
+                .deduplication()
+                .noCryptoConfig()
+                .disableSingleSave())
+            .build())
+        .extension(new DockerOpenDistroExtension(DockerOpenDistroSingleton.INSTANCE))
+        .extension(new CassandraExtension())
+        .extension(new RabbitMQExtension())
+        .extension(new AwsS3BlobStoreExtension())
+        .server(configuration -> DistributedServer.createServer(configuration)
+            .overrideWith(new LinagoraTestJMAPServerModule())
```

Comment:
Do we need this?

Reply:
I copied it from another sample, I tried to remove it but it was born another error, take time to fix it. I think it is not a problem?

Reply:
Ok so let just skip it